src/spaces/utilities.jl

```diff
@@ -73,19 +73,20 @@ periodicity of the space.
 get_direction(from, to, model::ABM) = get_direction(from, to, model.space)
 # Periodic spaces version
 function get_direction(
-    from::NTuple{D,Float64},
-    to::NTuple{D,Float64},
+    from::ValidPos,
+    to::ValidPos,
     space::Union{ContinuousSpace{D,true},AbstractGridSpace{D,true}}
 ) where {D}
-    best = to .- from
-    for offset in Iterators.product([-1:1 for _ in 1:D]...)
-        dir = to .+ offset .* spacesize(space) .- from
-        sum(dir.^2) < sum(best.^2) && (best = dir)
-    end
-    return best
+    direct_dir = to .- from
+    inverse_dir = direct_dir .- sign.(direct_dir) .* spacesize(space)
+    return map((x, y) -> abs(x) < abs(y) ? x : y, direct_dir, inverse_dir)
```

Comment:
```suggestion
    return map((x, y) -> minimum(abs, (x, y)), direct_dir, inverse_dir)
```
I still think there's a better way of writing this than this but Julia syntax says no.

Reply:
Yeah what I need is [this](https://mlochbaum.github.io/BQN/doc/compose.html#over) from BQN, oh well :')

Reply:
wait, it doesn't work since it applies abs to the result!

Reply:
I don't think that's correct:
```julia
names = "Julion", "Julia", "Zule"
minimum(length, names) # 4
length(minimum(names)) # 5
```

Reply:
that's indeed what I mean, when you do minimum(abs, (x,y)) you apply the abs to the iterable, in my implementation instead the returned value is x or y itself not the abs, abs is just used for comparison

Reply:
Oh wow I'm silly for missing this...

Reply:
I'd like a more readable version but looking in the docs I don't find a way to do this, this should be possible though, probably we should change base!

Reply:
```suggestion
    return map((x, y) -> argmin(abs, (x, y)), direct_dir, inverse_dir)
```

Reply:
right!! thank you, actually I saw this, but I thought about applying it to `direct_dir` and `inverse_dir` directly and then I gave up xD. 

still I think something like this should work in julia: `argmin.(abs, direct_dir, inverse_dir)`